pymc3/step_methods/mlda.py

```diff
@@ -356,10 +356,6 @@ class MLDA(ArrayStepShared):
     default_blocked = True
     generates_stats = True
 
-    # stat data types are different, depending on the base sampler.
-    # these are assigned in the init method.
-    stats_dtypes = None
-
```

Comment:
This is overwritten anyway, might as well set it in `pymc3/step_methods/arraystep.py`